Documentation/docs/learn/python_quick_start.md

```diff
@@ -0,0 +1,495 @@
+
+
+# Do interesting things...
+
+
+# Convert back to itk mesh data structure
+itk_mesh = itk.mesh_from_dict(mesh_dict)
+
+# Save result
+itk.meshwrite(itk_mesh, output_mesh_filename)
+```
+
+NumPy and `itk.Transform`:
+
+```
+# Read input transforms
+#
+# This is a Python list
+#
+# When there is more than one transformation
+# the list defines a transformation chain
+itk_transforms = itk.transformread(input_transform_filename)
+
+# Convert to standard Python data types + NumPy arrays
+transform_dicts = [itk.dict_from_transform(t) for t in itk_transforms]
+
+
+# Do interesting things...
+
+
+# Convert back to itk transform instance
+itk_transforms = [itk.transform_from_dict(t) for t in transform_dicts]
+
+# Save result
+itk.transformwrite(itk_transforms, output_transform_filename)
+```
+
+NumPy and `itk.Matrix`, VNL vectors, and VNL matrices:
+
+```py
+# VNL matrix from np.ndarray
+arr = np.zeros([3,3], np.uint8)
+matrix = itk.vnl_matrix_from_array(arr)
+
+# Array from VNL matrix
+arr = itk.array_from_vnl_matrix(matrix)
+
+# VNL vector from np.ndarray
+vec = np.zeros([3], np.uint8)
+vnl_vector = itk.vnl_vector_from_array(vec)
+
+# Array from VNL vector
+vec = itk.array_from_vnl_vector(vnl_vector)
+
+# itk.Matrix from np.ndarray
+mat = itk.matrix_from_array(np.eye(3))
+
+# np.ndarray from itk.Matrix
+arr = itk.array_from_matrix(mat)
+# Equivalent
+arr = np.asarray(mat)
+```
+
+### ITK and ITK-Wasm
+
+[ITK-Wasm](https://wasm.itk.org) can be used with native `itk` Python bindings.
+
+Both packages support common Python dictionary representations of the data structures used on interfaces. The non-dictionary types are more convenient to work with directly and provide strong typing for function calls.
+
+## Convert from `itkwasm` to `itk`
+
+To convert from an `itkwasm` dataclass interface type to a native `itk` Python type, first convert the `itkwasm` type to a dictionary, then use the `itk.<type>_from_dict` function. Example:
+
+```py
+from itkwasm import Image
+from dataclasses import asdict
+itkwasm_image = Image()
+image_dict = asdict(itkwasm_image)
+
+import itk
+itk_image = itk.image_from_dict(image_dict)
+```
+
+## Convert from `itk` to `itkwasm`
+
+To convert from a native `itk` Python type to an `itkwasm` dataclass interface type, first convert the `itkwasm` type to a dictionary the `itk.<type>_from_dict`, then pass the dictionary as keyword arguments to `itkwasm` constructor with the `**` operator. Example:
+
+
+```py
+import itk
+# Create an itk.Image
+itk_image = itk.Image.New()
+itk_image.SetRegions([8,8])
+itk_image.Allocate()
+image_dict = itk.dict_from_image(itk_image)
+
+from itkwasm import Image
+itkwasm_image = Image(**image_dict)
+```
+
+## itkwasm file formats
+
+`itkwasm` provides file formats corresponding to its interface types. These file formats keep wasm module sizes tiny, enable efficient and one-to-one serialization, assist with debugging, and bridge with [Web3 technologies](https://en.wikipedia.org/wiki/Web3).
```

Comment:
(...) keep wasm module (...) : (...) keep Wasm module (...) ?

According to https://webassembly.org/